frontend/public/components/_storage-form.scss

```diff
@@ -0,0 +1,7 @@
+.create-storage-class-form__subsection {
+  margin-left: 40px;
```

Comment:
The indentation feels a little strange to me. Would a horizontal separator and/or subheading be better?

Reply:
I like the indentation as it clearly indicates that the settings there belong to the item above it, but I'm open to changing it. @cshinn, care to weigh in here?

Reply:
@spadgett The indentation was based on the PF guidelines for progressive disclosure available [here ](https://www.patternfly.org/pattern-library/forms-and-controls/progressive-disclosure/) That said, I think a smaller indentation (maybe 20px or so) would get the job done and might not be as jarring

Reply:
I've updated the indentation to 20px. I think it looks good.

![okd - google chrome_514](https://user-images.githubusercontent.com/8544299/47033252-52ae4700-d142-11e8-856e-1a47d23d021b.png)